manager/manager.go

```diff
@@ -364,6 +367,20 @@ func (self *manager) containerDataSliceToContainerInfoSlice(containers []*contai
 	return output, nil
 }
 
+func (self *manager) GetContainerDerivedStats(containerName string) (info.DerivedStats, error) {
+	var ok bool
+	var cont *containerData
+	func() {
```

Comment:
+1. I was thinking the same thing.

On Wed, Feb 18, 2015 at 10:33 AM, Victor Marmol <notifications@github.com>
wrote:

> In manager/manager.go
> <https://github.com/google/cadvisor/pull/515#discussion_r24926758>:
>
> > @@ -364,6 +367,20 @@ func (self *manager) containerDataSliceToContainerInfoSlice(containers []*contai
> >  	return output, nil
> >  }
> >
> > +func (self *manager) GetContainerDerivedStats(containerName string) (info.DerivedStats, error) {
> > +	var ok bool
> > +	var cont *containerData
> > +	func() {
>
> unrelated and not for this PR: we do this everywhere, we should get a
> helper...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/515/files#r24926758>.
>